.github/actions/verify-token/src/predicate.ts

```diff
@@ -194,12 +206,6 @@ export function createPredicate(
     predicate.buildDefinition.systemParameters.GITHUB_EVENT_PAYLOAD = ghEvent;
   }
 
-  // Add workflow inputs to top-level externalParameters.
```

Comment:
These may belong somewhere else in `systemParameters` (as trusted env parameters from GH context) but it's not critical so let's defer.